### PR TITLE
MINIFICPP-17: Move processors to standard-processors in preparation for

### PR DESCRIPTION
**Pull request by phrocker**
future changes.

Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by phrocker** [comment]
No interfaces or API functions should change here.

File: cmake/BuildTests.cmake [hunk 1]
```diff
@@ -135,9 +135,25 @@ FOREACH(testfile ${NANOFI_UNIT_TESTS})
...
+    target_include_directories(${testfilename} BEFORE PRIVATE "${CMAKE_SOURCE_DIR}/extensions/standard-processors/")
+    target_include_directories(${testfilename} BEFORE PRIVATE "${CMAKE_SOURCE_DIR}/extensions/standard-processors/processors/")
     target_include_directories(${testfilename} BEFORE PRIVATE "${CMAKE_SOURCE_DIR}/libminifi/test")
     appendIncludes("${testfilename}")
     target_link_libraries(${testfilename} ${CMAKE_THREAD_LIBS_INIT} ${CATCH_MAIN_LIB} nanofi)
+    
+    if (APPLE)
+    # minifi-standard-processors
+	target_link_libraries (${testfilename} -Wl,-all_load minifi-standard-processors nanofi)
+	elseif(NOT WIN32)
```
Comment (arpadboda) [inline cmake/BuildTests.cmake]: Could you fix indentation here?

File: extensions/coap/tests/CMakeLists.txt [hunk 2]
```diff
@@ -28,6 +28,7 @@ FOREACH(testfile ${CURL_INTEGRATION_TESTS})
   	add_executable("${testfilename}" "${testfile}")
   	target_include_directories(${testfilename} BEFORE PRIVATE ${COAP_INCLUDE_DIRS})
   	target_include_directories(${testfilename} BEFORE PRIVATE ${CURL_INCLUDE_DIRS})
+  	target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/standard-processors")
```
Comment (arpadboda) [inline extensions/coap/tests/CMakeLists.txt]: Why different order of "private" and "before" here?

Comment (phrocker) [reply extensions/coap/tests/CMakeLists.txt]: I copied and pasted it. Good catch!

**Comment by phrocker** [comment]
@arpadboda the nanofi examples function as they did prior to this pull request

**Comment by arpadboda** [comment]
> @arpadboda the nanofi examples function as they did prior to this pull request

Cool, thanks for verifying, approved!

**Pull request closed without merging**